document/FileFormat.mdk

```diff
@@ -2629,6 +2629,14 @@ The tool might provide a "related location" (&sect;[#result-relatedLocations]) w
 an `annotatedCodeLocation` object with the message "Insecure data entered the system here". 
 ~
 
+### `callee` property { #annotatedCodeLocation-callee }
+
+If the value of the `kind` property (&sect;[#annotatedCodeLocation-kind]) of this `annotatedCodeLocation` object
```

Comment:
as per 'fullyQualifiedName', we should note that this value could be a key to acquire a logical location from that construct. for completeness, note that the format does not provide key collision resolution, if SARIF producers generate a collision and still insist on expressing caller/callee as logical locations.

per format precedent, the way to resolve would be to introduce a 'callerKey' and 'calleeKey' member. I would prefer if we can think of a rationale to avoid adding them.